ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/EntityUtil.java

```diff
@@ -42,10 +42,6 @@ private EntityUtil() {
   }
 
   private static List<FieldInfo> getFields(final List<Field> fields, final String type) {
-    if (fields.isEmpty()) {
```

Comment:
Likewise, I don't think we should be doing this either...

Reply:
Same as above